core/src/main/scala-2.13+/cats/data/NonEmptyLazyList.scala

```diff
@@ -110,7 +110,7 @@ class NonEmptyLazyListOps[A](private val value: NonEmptyLazyList[A])
    * Returns a new NonEmptyLazyList consisting of `a` followed by this
    */
   final def prepend[AA >: A](a: AA): NonEmptyLazyList[AA] =
-    create(a #:: toLazyList)
+    create(toLazyList.prepended(a))
```

Comment:
`#::` for `LazyList` is by-name, but `a` here is not, so it's wasteful/pointless to call it